decidim-core/lib/decidim/menu.rb

```diff
@@ -14,6 +14,8 @@ def initialize(name)
     # Evaluates the registered configurations for this menu in a view context
     #
     def build_for(context)
+      return if registry.nil?
```

Comment:
Why this change?

Reply:
The first version of the patch was requiring this change because might happen that user_menu was empty. Without this condition decidim was hanging. We may remove it since after last change the user menu comes populated with the default options.

Reply:
Then I say let's remove it! :)